quadkey/bq/sql/ST_ASQUADINTPOLYFILL.sql

```diff
@@ -0,0 +1,28 @@
+-----------------------------------------------------------------------
+--
+-- Copyright (C) 2021 CARTO
+--
+-----------------------------------------------------------------------
+
+CREATE OR REPLACE FUNCTION `@@BQ_PROJECTID@@.@@BQ_DATASET_QUADKEY@@.POLYFILL_FROMGEOJSON`
```

Comment:
I understand that this function is private right? In that case I would name it as __ST_ASQUADINT_POLYFILL